src/perf/lib/SecNetPerfMain.cpp

```diff
@@ -245,19 +249,26 @@ QuicMainStart(
 #ifndef _KERNEL_MODE
```

Comment:
@nibanks is there a reason the code block is behind `_KERNEL_MODE` rather than `QUIC_USE_RAW_DATAPATH `?

Reply:
Because of `std::vector` isn't available in kernel.

Reply:
I changed it from an on stack array to a vector in this PR. The code before this PR uses `_KERNEL_MODE`.

Reply:
Ah, you're right. It was because I never expected XDP to be used in kernel, but other user mode variants might.